### PR TITLE
templates: addition of invenio font awesome

### PR DESCRIPTION
**Pull request by JavierDelgadoFernandez**
- Adds font awesome with orcid and CERN logo.

Signed-off-by: Javier Delgado javier.delgado.fernandez@cern.ch

### CODE REVIEW

**Comment by JavierDelgadoFernandez** [comment]
The fonts contain two ```Generated by (...)``` one from the tool used and another one from the frontend used. I didn't remove this message.

**Comment by lnielsen** [comment]
Where did you get the ORCID logo from. We will for sure we need to add a special comment in the license file that the logo is trademarked by ORCID.

See also https://orcid.org/trademark-and-id-display-guidelines

**Comment by JavierDelgadoFernandez** [comment]
@lnielsen Due to they are not providing a vectorial image, I created it manually. If you plan to add to invenio stable, you will need to ask for a real one... The sizes of the items and the font are correct but probably the position of the letters is not very accurate...

**Comment by lnielsen** [comment]
I will request the SVG (http://support.orcid.org/forums/175591-orcid-ideas-forum/suggestions/6731319-orcid-icons-format)

**Comment by lnielsen** [comment]
I have a support request open at ORCID. Hope to have the answer today.

**Comment by lnielsen** [comment]
I got the SVG logo. We however won't be able to include it as a font due to color requirements of the logo.

**Comment by jirikuncar** [comment]
@lnielsen we should be able to set the same color as `font-color` in css if color is the problem.

**Comment by lnielsen** [comment]
The problem is to use the logo in any other color than the ORCID green.

**Comment by jirikuncar** [comment]
@lnielsen this wouldn't work
```css
.fa.fa-orcid {
  color: "green"; !important  /* Do not remove this rule as it is legally required by ORCID. */
}
```

**Comment by lnielsen** [comment]
@jirikuncar If it works I guess it's ok. Javier I've sent you the SVG.

**Comment by lnielsen** [comment]
Color code #A6CE39

**Comment by JavierDelgadoFernandez** [comment]
@lnielsen @jirikuncar I have updated the PR with the SVG logo and the color. Here is a screenshot with the current login page:
orcid_logo

**Comment by lnielsen** [comment]
The "ID" in ORCID logo looks misaligned. Compare with these: 
orcid_16x16
orcid_24x24

File: invenio_theme/static/scss/invenio_theme/invenio-font-awesome.scss [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    font-family: 'invenio font awesome';
+    src: url('#{$invenio-font-awesome-path}/invenio-font-awesome.eot');
+    src: url('#{$invenio-font-awesome-path}/invenio-font-awesome.eot?#iefix') format('embedded-opentype'),
+         url('#{$invenio-font-awesome-path}/invenio-font-awesome.woff') format('woff'),
+         url('#{$invenio-font-awesome-path}/invenio-font-awesome.ttf') format('truetype'),
+         url('#{$invenio-font-awesome-path}/invenio-font-awesome.svg#invenio-font-awesome') format('svg');
+    font-weight: normal;
+    font-style: normal;
+}
+[class*='fa-']:before{
```
Comment (jirikuncar) [inline invenio_theme/static/scss/invenio_theme/invenio-font-awesome.scss]: IMHO only `font-family` should be needed and only for `fa-cern` and `fa-orcid`

**Comment by lnielsen** [comment]
@JavierDelgadoFernandez I managed to fix your issue, and have sent you a version of the font. There's  however another issue using the font. The ID part has to be transparent in the logo, which means that it will get a wrong color (i.e. not white) depending on which background you use it on. As it is now, I don't think it's worthwhile continuing trying to create a font out of it.

**Pull request closed without merging**